content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -1168,6 +1168,11 @@ Any
 Most recent
 : Return data at the most recent resource version. The returned data must be
   consistent (in detail: served from etcd via a quorum read).
+  For etcd v3.4.31+ and v3.5.13+ Kubernetes {{< skew currentVersion >}} serves “most recent” reads from the _watch cache_:
+  an internal, in-memory store within the API server that caches and mirrors the state of data
+  persisted into etcd. Kubernetes requests progress notification to maintain cache consistency against
+  the etcd persistence layer. Kubernetes versions v1.28 through to v1.30 also supported this
```

Comment:
@sftim Don't know if we want to mentioned this feature pre v1.31, it was in alpha state and proven to be very buggy.

Reply:
Let's write something, even if we caveat it (eg to say that using the alpha feature is not recommended). Assume that the reader has never heard of a watch cache and is running an older version; if we are mute on the topic, they might enable the (alpha) feature gate in their cluster and see what happens.

Reply:
Done